pkg/bff/members.go

```diff
@@ -122,12 +140,50 @@ func (s *Server) Overview(c *gin.Context) {
 		out.TestNet.Vasps = int(testnet.Vasps)
 		out.TestNet.CertificatesIssued = int(testnet.CertificatesIssued)
 		out.TestNet.NewMembers = int(testnet.NewMembers)
+
+		if testnetID != "" {
+			// Check if we received the VASP details from the testnet
+			if testnet.Vasp == nil {
+				log.Error().Msg("expected VASP details from testnet Summary RPC")
+				c.JSON(http.StatusInternalServerError, api.ErrorResponse(fmt.Errorf("could not retrieve testnet VASP details")))
+				return
+			}
+
+			out.TestNet.MemberDetails = api.MemberDetails{
+				ID:          testnet.Vasp.Id,
+				Status:      testnet.Vasp.Status.String(),
+				CountryCode: testnet.Vasp.Country,
+			}
+		} else {
+			out.TestNet.MemberDetails = api.MemberDetails{
+				Status: pb.VerificationState_NO_VERIFICATION.String(),
+			}
```

Comment:
I guess the frontend will have to check for this status to determine existence, should we make it more explicit?

Reply:
I think it might be better to just leave the `MemberDetails` as `nil` and then we'll have the front-end check for `null` before populating the component. @masskoder thoughts?